content/en/tracing/metrics/runtime_metrics/_index.md

```diff
@@ -4,14 +4,216 @@ type: multi-code-lang
 aliases:
   - /tracing/advanced/runtime_metrics/
   - /tracing/runtime_metrics/
+  - /tracing/runtime_metrics/dotnet
+  - /tracing/runtime_metrics/go
+  - /tracing/runtime_metrics/java
+  - /tracing/runtime_metrics/nodejs
+  - /tracing/runtime_metrics/python
+  - /tracing/runtime_metrics/ruby
 description: "Gain additional insights into an application's performance with the runtime metrics associated to your traces."
+further_reading:
+    - link: 'tracing/other_telemetry/connect_logs_and_traces'
+      tag: 'Documentation'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/trace_collection/custom_instrumentation'
+      tag: 'Documentation'
+      text: 'Manually instrument your application to create traces.'
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
 ---
 
+Runtime metrics are application metrics about memory usage, garbage collection, or parallelization. Datadog tracing libraries can automatically collect these metrics for supported environments and send them to the Datadog agent.
+
+## Tracing requirements
+
+| Language | Library Version       | Support Level | Enabled By Default | Generates runtime-id granularity |
+|----------|-----------------------|---------------|--------------------|----------------------------------|
+| Java     | 0.29.0+               | GA            | Yes                | Yes                              |
+| Go       | 1.18.0+               | GA            | No                 | Yes                              |
+| .NET     | 1.23.0+               | GA            | No                 | Yes                              |
+| Node.js  | 3.0.0+                | GA            | No                 | No                               |
+| Ruby     | 0.44.0+               | GA            | No                 | No                               |
+| Python   | 0.30.0+               | Preview       | No                 | No                               |
+| PHP      | Not supported         |               |                    |                                  |
+| C++      | Not supported         |               |                    |                                  |
+
+### Caveats
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+#### Supported runtimes
+Runtime metrics are only supported on Java 8.
+
+{{% /tab %}}
+
+{{% tab "Ruby" %}}
+
+You must add the [`dogstatsd-ruby`][1] gem to your Ruby application.
```

Comment:
For some reason this link isn't rendering properly
<img width="533" alt="image" src="https://github.com/user-attachments/assets/978b3f79-e9fd-4743-8dc6-347c9d9e7e29" />